deezer_importer.user.js

```diff
@@ -0,0 +1,134 @@
+// ==UserScript==
+// @name           Import Deezer releases into MusicBrainz
+// @namespace      https://github.com/murdos/musicbrainz-userscripts/
+// @description    One-click importing of releases from deezer.com into MusicBrainz
+// @version        2019.1.30.1
+// @downloadURL    https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/deezer_importer.user.js
+// @updateURL      https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/deezer_importer.user.js
+// @include        http*://www.deezer.com/*/album/*
+// @require        https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js
+// @require        lib/mbimport.js
+// @require        lib/logger.js
+// @require        lib/mbimportstyle.js
+// @icon           https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/assets/images/Musicbrainz_import_logo.png
+// ==/UserScript==
+
+// prevent JQuery conflicts, see http://wiki.greasespot.net/@grant
+this.$ = this.jQuery = jQuery.noConflict(true);
+
+$(document).ready(function() {
+    MBImportStyle();
+    let releaseUrl = window.location.href.replace(/\?.*$/, '').replace(/#.*$/, '');
+    let releaseId = releaseUrl.replace(/^https?:\/\/www\.deezer\.com\/[^/]+\/album\//i, '');
+    let deezerApiUrl = `https://api.deezer.com/album/${releaseId}`;
+
+    $.ajax({
+        url: deezerApiUrl,
+        dataType: 'jsonp',
```

Comment:
Is it necessary to use JSONP?

Reply:
The Deezer API doesn't send a Access-Control-Allow-Origin header so the requests were being blocked until I switched to JSONP.